packages/auth/src/features/authenticate/handler.test.ts

```diff
@@ -78,14 +79,18 @@ describe('authenticate handler receives a request', () => {
 
       jest.spyOn(reloadedCodeService, 'generateNonce').mockReturnValue('12345')
 
-      fetch.mockResponse(
+      fetch.mockResponseOnce(
         JSON.stringify({
-          userId: '1',
+          id: '1',
           status: 'active',
-          scope: ['admin'],
+          role: 'NATIONAL_SYSTEM_ADMIN',
```

Comment:
`user-mgnt` doesn't return scopes anymore as it doesn't know what scopes are. Scopes are now controlled by CC's `/roles` endpoint in combination with the auth service which creates JWT tokens.